src/Couchbase.Lite.Shared/API/Query/DataSource.cs

```diff
@@ -39,10 +40,25 @@ public static class DataSource
 
         /// <summary>
         /// Creates a data source for an <see cref="IQuery" /> that gets results from the given
+        /// <see cref="Collection(Lite.Collection)" />
+        /// </summary>
+        /// <param name="collection">The collection to operate on</param>
+        /// <returns>The source of data for the <see cref="IQuery" /></returns>
+        [NotNull]
+        public static IDataSourceAs Collection([NotNull] Collection collection)
```

Comment:
ICollection

Reply:
Please see below TODO commentsfrom line 51 to line 53

Reply:
I don't see any todo comments in this file